Sources/Common/DataModel/ITKHelper/index.js

```diff
@@ -0,0 +1,29 @@
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+
+/**
+ * Converts an itk.js image to a vtk.js image.
+ *
+ * Requires an itk.js image as input.
+ */
+function convertItkToVtkImage(itkImage) {
+  // create VTK image data
+  const imageData = vtkImageData.newInstance({
+    origin: itkImage.origin.slice(),
+    spacing: itkImage.spacing.slice(),
+  });
+  const scalars = vtkDataArray.newInstance({
+    values: itkImage.data,
+    numberOfComponents: itkImage.imageType.components,
```

Comment:
Do you have a name for that array?

Reply:
No....."Scalars"?

Reply:
Then add `name: 'Scalars',1

Reply:
Maybe ask for it as 2nd arg with a default to 'Scalars'

Reply:
Seems reasonable. Would it be preferable to accept that through options object as the 2nd arg in the event we ever decide to add more configurability to the conversion function?